tests/conftest.py

```diff
@@ -2887,3 +2901,126 @@ def fixture_demultiplexed_flow_cells_tmp_directory(tmp_path) -> Path:
     tmp_dir = Path(tmp_path, "tmp_run_dir")
 
     return Path(shutil.copytree(original_dir, tmp_dir))
+
+
+## Taxprofiler fixtures
+
+
+@pytest.fixture
+def taxprofiler_config(taxprofiler_dir: Path, taxprofiler_case_id: str) -> None:
+    """Create samplesheet.csv file for testing"""
+    Path.mkdir(Path(taxprofiler_dir, taxprofiler_case_id), parents=True, exist_ok=True)
+    Path(taxprofiler_dir, taxprofiler_case_id, f"{taxprofiler_case_id}_samplesheet.csv").touch(
+        exist_ok=True
+    )
+
+
+@pytest.fixture(name="taxprofiler_case_id")
+def fixture_taxprofiler_case_id() -> str:
+    """Returns a taxprofiler case id."""
+    return "taxprofiler_case"
+
+
+@pytest.fixture(name="taxprofiler_sample_id")
+def fixture_taxprofiler_sample_id() -> str:
+    """Returns a rnafusion sample id."""
+    return "taxprofiler_sample"
+
+
+@pytest.fixture(name="taxprofiler_dir")
+def taxprofiler_dir(tmpdir_factory, apps_dir: Path) -> str:
+    """Return the path to the taxprofiler apps dir."""
+    taxprofiler_dir = tmpdir_factory.mktemp("taxprofiler")
+    return Path(taxprofiler_dir).absolute().as_posix()
+
+
+@pytest.fixture(name="taxprofiler_housekeeper_dir")
+def fixture_taxprofiler_housekeeper_dir(tmpdir_factory, taxprofiler_dir: Path) -> Path:
+    """Return the path to the taxprofiler housekeeper bundle dir."""
+    return tmpdir_factory.mktemp("bundles")
+
+
+@pytest.fixture(name="taxprofiler_fastq_file_l_1_r_1")
```

Comment:
since we are repeating fixtures I wonder if we can rename them and use them in all analyses that need them (balsamic, rnafusion and. taxprofiler for now)

Reply:
same for:
- fixture_rnafusion_fastq_file_l_1_r_2 
- taxprofiler_case_id
- taxprofiler_sample_id
- taxprofiler_dir
- taxprofiler_housekeeper_dir
- taxprofiler_mock_fastq_files
and all the ones that are the same between them

Reply:
I am thinking if we can do it in a separate PR. In that case I can create a separate issue.

Reply:
Sure NP 👍